src/storage/admin/AdminTaskManager.cpp

```diff
@@ -25,10 +26,83 @@ bool AdminTaskManager::init() {
 
   bgThread_->addTask(&AdminTaskManager::schedule, this);
   shutdown_ = false;
+  handleUnreportedTasks();
   LOG(INFO) << "exit AdminTaskManager::init()";
   return true;
 }
 
+void AdminTaskManager::handleUnreportedTasks() {
+  if (env_ == nullptr) return;
+  unreportedAdminThread_.reset(new std::thread([this] {
+    bool ifAny = true;
+    while (ifAny) {
+      ifAny = false;
```

Comment:
do {} while(true). Use break to stop the loop .

Reply:
> do {} while(true). Use break to stop the loop .
Are there any rules stipulate that we shoul use do {} while(true) here? I don't think it's a good idea.